examples/debug/carto_vl/style-guide.ipynb

```diff
@@ -0,0 +1,2382 @@
+       "    }\n",
+       "\n",
+       "    function setPopupsClick(tempPopup, interactivity, popupHeader, values) {\n",
+       "      interactivity.off('featureHover', (event) => {\n",
+       "          updatePopup(tempPopup, event, popupHeader, values)\n",
+       "      });\n",
+       "      interactivity.on('featureClick', (event) => {\n",
+       "          updatePopup(tempPopup, event, popupHeader, values)\n",
+       "      });\n",
+       "    }\n",
+       "    \n",
+       "    function setPopupsHover(tempPopup, interactivity, popupHeader, values) {\n",
+       "      interactivity.off('featureClick', (event) => {\n",
+       "          updatePopup(tempPopup, event, popupHeader, values)\n",
+       "      });\n",
+       "      interactivity.on('featureHover', (event) => {\n",
+       "          updatePopup(tempPopup, event, popupHeader, values)\n",
+       "      });\n",
+       "    }\n",
+       "\n",
+       "    function updatePopup(layer_popup, event, popupHeader, values) {\n",
+       "      if (event.features.length > 0) {\n",
+       "        const feature = event.features[0];\n",
+       "        const variables = values && values.length\n",
+       "          ? Object.keys(feature.variables).filter(variable => values.include(variable))\n",
+       "          : Object.keys(feature.variables);\n",
+       "\n",
+       "        let popupHTML = popupHeader ? `<h1>${popupHeader}</h1>` : ``;\n",
+       "        \n",
+       "        Object.keys(feature.variables).forEach((name) => {\n",
+       "            const variable = feature.variables[name];\n",
+       "\n",
+       "            if (variable) {\n",
+       "              console.log('!!! variable', variable);\n",
+       "              popupHTML += `\n",
+       "                <h3 class=&quot;h3&quot;>${name}</h3>\n",
+       "                \n",
+       "              `;\n",
+       "            }\n",
+       "            // <p class=&quot;description open-sans&quot;>${JSON.stringify(variable.value)}</p>\n",
+       "        });\n",
+       "        \n",
+       "        layer_popup\n",
+       "            .setLngLat([event.coordinates.lng, event.coordinates.lat])\n",
+       "            .setHTML(`<div>${popupHTML}</div>`);\n",
+       "        \n",
+       "        if (!layer_popup.isOpen()) {\n",
+       "          layer_popup.addTo(map);\n",
+       "        }\n",
+       "      } else {\n",
+       "        layer_popup.remove();\n",
+       "      }\n",
+       "    }\n",
+       "  });\n",
+       "}\n",
+       "\n",
+       "responsive.addEventListener('ready', onReady);\n",
+       "    </script>\n",
+       "  </body>\n",
+       "  </html>\n",
+       "\">\n",
+       "\n",
+       "</iframe>"
+      ],
+      "text/plain": [
+       "<cartoframes.carto_vl.maps.map.Map at 0x116761940>"
+      ]
+     },
+     "execution_count": 20,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "## dictionary style\n",
```

Comment:
I rather prefer this over the array style. 💪 

Not so sure about the kebab-case, but not strong opinion here.

Reply:
@rochoa +1 to dictionary style i like that too :) regarding the kebab-case, that was because what i see with styling languages is that convention. the camel case, from the discussions,  is more convention between programming languages.

but now I'm thinking, do we need to do any of that since we don't need to add some sort of vl identifier to duplicate python properties like `filter` and `transform`

Reply:
+1 for dictionary style too!

I still feel against the `strokeColor` since it's not Pythonic, but other cross-language syntax things make it into Python (Scala's camelCase showing up in pyspark). I think people will be irritated with either one we choose, but I'm guessing less so for `strokeColor` because: 1. it's the same in CARTO VL so it's easy to translate/copy-paste; 2. it's easier to type that including the `-`.